docassemble/ALWeaver/data/questions/assembly_line.yml

```diff
@@ -238,26 +242,76 @@ help:
     There are ${len(all_fields)} fields total.
 
     `${all_fields}`
+
+---
+code: |
+  interview.country = 'US'
+  # This is weird but needed for check-in logic
+  # TODO(qs): adjust when we have a way to load in org defaults at runtime
+---
+event: update_state_list
+code: |
+  if action_argument('interview.country') and action_argument('interview.country') != interview.country:          
+    interview.country = action_argument('interview.country')
+    background_response('refresh')
+  background_response()    
 ---
 id: dependency question
-question: Which jurisdictions and organizations is this form a part of?
-subquestion: |
-  This will help the Weaver to add appropriate jurisdictions as dependencies for your interview. If you are unsure, leave the default options checked.
-help:
-  label: My jurisdiction or organization is not here
-  content: |
-    New jurisdictions will be added soon. You should add those dependencies manually in the Playground until then.
+question: |
+  Brand, Location and Identity
 fields:
-  - Jurisdictions: interview.jurisdiction_choices
+  - Author's name(s) (one per line): interview.author
+    datatype: area
+    rows: 3
+    # TODO(qs): update default if we have a way to load org defaults
+    default: |
+      % if user_logged_in():
+      ${ user_info().first_name } ${ user_info().last_name }
+      % else:
+      Court Forms Online
+      % endif
+    required: False
+    help: |
+      The information in this box will be public by default.
+      It will be used on GitHub and on the "About" page of the interview.
```

Comment:
It will be in a file (setup.py) that is published to GitHub. So in theory, someone could search and find your name on GitHub if the package gets published to GitHub.